tools/locales/de.json

```diff
@@ -1,7 +1,13 @@
 {
-    "completion": "Drucke shell Vervollständigungsskript",
-    "epilog": "Dokumentation",
```

Comment:
I can add a translation, still looking for a fix for the problem described by @kamilchm in QingWei-Li/docsify-cli@02dec13